Java/E-montazysta/src/main/java/com/emontazysta/service/impl/OrdersServiceImpl.java

```diff
@@ -45,12 +56,19 @@ public OrdersDto getById(Long id) {
     }
 
     @Override
+    @Transactional
     public OrdersDto add(OrdersDto ordersDto) {
 
         Orders order = ordersMapper.toEntity(ordersDto);
         order.setCreatedAt(LocalDateTime.now());
-        order.setEditedAt(null);
-        return ordersMapper.toDto(repository.save(order));
+        order.setEditedAt(LocalDateTime.now());
```

Comment:
```suggestion
        order.setEditedAt(null);
```
Przy tworzeniu czas edycji nie istnieje